.github/workflows/build-test-pr.yml

```diff
@@ -113,6 +123,53 @@ jobs:
         files: lib/msal-${{matrix.library}}/coverage/lcov.info
         flags: msal-${{ matrix.library }}
 
+  validation-lib-build-test:
```

Comment:
Did we make this a separate job because it doesn't have "msal" in the name? If so, I almost prefer updating the `lib-build-test` job to be more generic so we don't have to duplicate the logic

Reply:
Makes sense. Updated.